flink-runtime/src/test/java/org/apache/flink/runtime/dispatcher/DispatcherTest.java

```diff
@@ -629,7 +630,7 @@ public void testJobSubmissionErrorAfterJobRecovery() throws Exception {
 	}
 
 	/**
-	 * Tests that a blocking {@link JobManagerRunner} creation, e.g. due to blocking FileSystem access,
+	 * Tests that a blocking {@link JobManagerRunnerImpl} creation, e.g. due to blocking FileSystem access,
```

Comment:
I think it is modified by auto refactor tools. In document we can only document interface instead of implementation. Thus +1 replace this-alike `JobManagerRunnerImpl` in document with `JobManagerRunner`.

Reply:
Yes it should be `JobManagerRunner`. Will update.